.github/workflows/test-go.yaml

```
@@ -105,6 +105,31 @@ jobs:
       SLACK_G_HELP_ENGINEERING_WEBHOOK_URL: ${{ secrets.SLACK_G_HELP_ENGINEERING_WEBHOOK_URL }}
 
+  # ──────────────────────────────────────────────────────────────────────────
+  # MariaDB coverage: only on the nightly cron schedule.
+  #
+  # MariaDB is not a supported database. This leg exists to track how far from
+  # working it is without slowing down every push, so it is cron-only and
+  # continue-on-error. See https://github.com/fleetdm/fleet/issues/31288.
+  # ──────────────────────────────────────────────────────────────────────────
+  test-go-mariadb:
+    if: github.event_name == 'schedule'
+    continue-on-error: true
```

actionlint (1.7.12)

[error] when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "test-go-mariadb"

(syntax-check)